src/PaymentMethods/AbstractPaymentMethod.php

```diff
@@ -241,9 +250,9 @@ public function getMergedProperties(): array
     /**
      * Default values for the initial settings saved
      *
-     * @return array
+     * @return array|false
```

Comment:
Maybe would be more convenient to keep `array` and return an empty array if got `false`?

Or just `assert` array, I think it's not possible to get `false` here, `array_combine` returns it if

> the number of elements for each array is not equal

Reply:
Good point. I will keep the conditional, though, not sure if I will receive the same elements each time, just in case.